lib/mutant/repository.rb

```diff
@@ -20,17 +20,9 @@ def call(subject)
 
     # Diff between two objects in repository
     class Diff
-      include Adamantium, Concord.new(:from, :to)
+      include Adamantium, Anima.new(:config, :from, :to)
```

Comment:
I also base my decision also on: Is there a natural positional order, if not use anima, as arguments are named.

Here I could not imagine how I could infer the correct positioning of arguments if I just knew: "`Diff.new` takes 3 arguments".